internal/cluster/service_test.go

```diff
@@ -0,0 +1,257 @@
+package cluster
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	authClient "github.com/fabric8-services/fabric8-jenkins-idler/internal/auth/client"
+	openShiftClient "github.com/fabric8-services/fabric8-jenkins-idler/internal/openshift/client"
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/token"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_clusterService_GetClusterView(t *testing.T) {
```

Comment:
can't we use `TestClusterServiceGetClusterView`. I didn't see camel case mixed with snake case in standard go library?

Reply:
I auto-generate tests barebones using gotests and it introduced this convention from Example.
 https://github.com/cweill/gotests/issues/22.  
TestClusterServiceGetClusterView doesn't make any sense.  
1. Test_receiver's Name_method for Methods.
2. TestFuncName for Exported function.
3. Test_funcName for unexported function.

https://golang.org/pkg/testing/#hdr-Examples

Reply:
thanks, @khrm. I didn't know about this.